test/context.js

```diff
@@ -24,8 +24,8 @@ describe('context', function() {
     }
     zmq.Context.setMaxSockets(256);
     zmq.Context.getMaxSockets().should.equal(256);
-    zmq.Context.setMaxSockets(1024);
```

Comment:
@reqshark I'm not sure I follow. This file should be able to run fine without being aware what the "right" maxSockets is. I think what @ValYouW says makes sense.